docs/webpack.dev.config.js

```diff
@@ -16,6 +16,7 @@ const appPaths = [
 
 const config = decorateClientConfig({
   entry: [
+    'core-js/modules/es6.symbol',
```

Comment:
This was required for the site works in IE. It needed to be added before `react-hot-loader`, which is why it's in the `entries` array rather than `client-render.js` with the other polyfills.